Description: Do not ignore given CFLAGS
 The configure script ignores the CFLAGS provided by the user, because it sets
 the CFLAGS variable. This prevents us to set hardening flags.
Author: Benjamin Drung <benjamin.drung@profitbricks.com>

--- a/configure.ac
+++ b/configure.ac
@@ -271,7 +271,7 @@
 	AC_DEFINE([HAVE_PACKET_PACING], [1], [Have PACKET_PACING support])
 fi
 
-CFLAGS="-g -Wall -D_GNU_SOURCE -O3"
+CFLAGS="-g -Wall -D_GNU_SOURCE -O3 $CFLAGS"
 AC_SUBST([LIBUMAD])
 AC_SUBST([LIBMATH])
 AC_CONFIG_FILES([Makefile])
